repos/system_upgrade/el8toel9/actors/checkvdo/libraries/checkvdo.py

```diff
@@ -0,0 +1,30 @@
+from leapp import reporting
+
+_report_title = reporting.Title('VDO devices migration to lvm-based management')
+
+def check_vdo(conversion_info):
+    for pre_conversion in conversion_info.pre_conversion_vdos:
+        reporting.create_report([
+            _report_title,
+            reporting.Summary(
+                "VDO device '{0}' requires migration".format(
+                    pre_conversion.name)),
+            reporting.Severity(reporting.Severity.HIGH),
+            reporting.Tags([reporting.Tags.SERVICES, reporting.Tags.DRIVERS]),
+            reporting.Remediation(hint = 'Perform VDO to LVM migration for the VDO device.'),
```

Comment:
There will be.  I asked for a link I could include but the doc is not yet done.

Reply:
@jshimkus-rh in case the doc will exist for sure before the GA, we suggest to craft a shortened URL as has been done in couple of other actors recently, see e.g.: https://github.com/oamg/leapp-repository/pull/729 - where you can find the URL: https://red.ht/upgrading-RHEL-8-to-RHEL-9-network-scripts - which points to nowhere now. But we know the document will be created before the release, so it's ok, they will set the proper target link for this URL.